x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -351,6 +358,20 @@ const UnstyledProcessEventDot = React.memo(
               height={markerSize * 1.5}
               className="backing"
             />
+            {isOrigin && (
+              <use
+                xlinkHref={`#${SymbolIds.processCubeActiveBacking}`}
+                fill="transparent" // Transparent so we don't double up on the default hover
```

Comment:
:information_source: Does this make it through the `use` shadow boundary? This is something I have to look up again and again every time it comes up: https://tympanus.net/codrops/2015/07/16/styling-svg-use-content-css/